src/components/extractFields.js

```diff
@@ -38,6 +38,14 @@ function getLon(properties) {
   }
 }
 
+function getBoundingBox(properties) {
+  if (properties['lbl:bbox']) {
```

Comment:
Hm I think you're right, as well as for the other helper functions. For lat/lon for example, I bet we don't handle null island very well.